docs/jwst/jump/description.rst

```diff
@@ -60,6 +60,15 @@ The two-point difference method is applied to each integration as follows:
 * If there are only two differences (three groups), the smallest one is compared to the larger
   one and if the larger one is above a threshold, it is flagged as a jump.
 
+* If flagging of the 4 neighbors is requested, then the 4 adjacent pixels will
+  have ramp jumps flagged in the same group as the central pixel as long as it has
+  a jump between the min and max requested levels for this option.
+
+* If flagging of groups after a ramp jump is requested, then the groups in the
+  requested time since a detected ramp jump will be flagged as ramp jumps if
+  the ramp jump is above the requested threshold.  Two thresholds and times are
```

Comment:
what does 'two threshold and times' mean here?

Reply:
i figured this out based on context, but rephrase since you're not using 'time' anymore

Reply:
Yes, the user is still giving the argument values in units of time, but then those are converted into the equivalent number of groups down in the lower-level functions.

Reply:
ok, i guess i don't see the benefit of supplying time over just giving group number, but if this is more intuitive to users then thats fine

Reply:
Giving it in time means that we don't need to supply different values of the number of groups to flag for different subarrays.  The transient seen after the ramp jumps is seen and expected to be time dependent only.  Hence, this make sense and makes it easier to implement.